apps/explorer/src/state/network/const.ts

```diff
@@ -0,0 +1,5 @@
+import { CHAIN_INFO_ARRAY } from '@cowprotocol/common-const'
+
+export const NETWORKS_PREFIXES = CHAIN_INFO_ARRAY.map((info) => info.urlAlias)
```

Comment:
Networks on plural feels weird, I'm in favour of keeping in singular

```suggestion
export const NETWORK_PREFIXES = CHAIN_INFO_ARRAY.map((info) => info.urlAlias)
```